health/health.go

```diff
@@ -87,19 +89,16 @@ const (
 	SysTKA = Subsystem("tailnet-lock")
 )
 
-// NewWarnable returns a new warnable item that the caller can mark
-// as health or in warning state.
-func (t *Tracker) NewWarnable(opts ...WarnableOpt) *Warnable {
+// NewWarnable returns a new warnable item that the caller can mark as health or
+// in warning state via Tracker.SetWarnable.
+//
+// NewWarnable is generally called in init and stored in a package global. It
+// can be used by multiple Trackers.
+func NewWarnable(opts ...WarnableOpt) *Warnable {
```

Comment:
How about having this return a WarnableHandle? It's not clear that the pointer to a warnable object is being used as a key